deploy-example-jars/Dockerfile

```diff
@@ -17,7 +17,7 @@ VOLUME /tmp
 
 ARG K8_VERSION=v1.15.3
 
-RUN apk --no-cache add curl=7.69.1-r1
+RUN apk --no-cache add curl=7.69.1-r3
```

Comment:
Just a question.  It looks like we are updating the version for curl.  Is there any specific reason for this update?

Reply:
Mostly just because it stopped working...
It may just be that the base image is now shipping with curl `7.69.1-r3` which clashes with our added install. Either way, bumping the version number to what the error suggests fixes it

Reply:
OK